tests/functional/tests.py

```diff
@@ -1883,13 +1883,26 @@ def test_upload_snowball_objects(log_entry):
                     mod_time=datetime.now(),
                 ),
             ],
+            staging_filename=staging_filename
         )
         _test_list_objects_api(bucket_name, 3)
     finally:
         _CLIENT.remove_object(bucket_name, "my-object1")
         _CLIENT.remove_object(bucket_name, "my-object2")
         _CLIENT.remove_object(bucket_name, "my-object3")
         _CLIENT.remove_bucket(bucket_name)
+        if staging_filename and os.path.exists(staging_filename):
+            os.remove(staging_filename)
+
+def test_upload_snowball_objects(log_entry):
+    """Test upload_snowball_objects()."""
+    _test_upload_snowball_objects(log_entry)
+
+
+def test_upload_snowball_objects_with_staging(log_entry):
```

Comment:
Add in-line comment `  # pylint: disable=invalid-name`  to ignore this pylint error.